async_service/base.py

```diff
@@ -321,7 +328,7 @@ async def _run_and_manage_task(self, task: TaskAPI) -> None:
             self.logger.debug("%s: task %s raised CancelledError.", self, task)
             raise
         except Exception as err:
-            self.logger.debug(
+            self.logger.error(
```

Comment:
I'm unsure about logging this at error level since the exception will get re-raised upon service exit via a `MultiError`.  I'm interested in your reasoning for this.

Reply:
Hm, I don't remember the exact scenario, but there was an error that was getting swallowed into the debug logs. I only saw it when I flipped on this logging.